test/parallel/test-tls-over-http-tunnel.js

```diff
@@ -41,7 +41,8 @@ var proxy = net.createServer(function(clientSocket) {
       // Verify the CONNECT request
       assert.equal('CONNECT localhost:' + common.PORT + ' HTTP/1.1\r\n' +
                    'Proxy-Connections: keep-alive\r\n' +
-                   'Host: localhost:' + proxyPort + '\r\n\r\n',
+                   'Host: localhost:' + proxyPort + '\r\n' +
+                   'Connection: close\r\n\r\n',
```

Comment:
i'd say that if you're going to do that you should remove the `Proxy-Connections` header also instead of just leaving it there -- but another maintainer might want this change in a separate pull request anyways